src/components/MediaViewer/MediaViewer.js

```diff
@@ -4,15 +4,15 @@ import { HeaderBackButton } from "@react-navigation/elements";
 import type { Node } from "react";
 import React, { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
-import { Dimensions, Image } from "react-native";
+import { Dimensions, Image, View } from "react-native";
 import ImageZoom from "react-native-image-pan-zoom";
 import { Appbar, Button } from "react-native-paper";
 
 import Photo from "../../models/Photo";
 import { imageStyles, viewStyles } from "../../styles/mediaViewer/mediaViewer";
 import DeletePhotoDialog from "../SharedComponents/DeletePhotoDialog";
 import PhotoCarousel from "../SharedComponents/PhotoCarousel";
-import ViewNoFooter from "../SharedComponents/ViewNoFooter";
+// import ViewNoFooter from "../SharedComponents/ViewNoFooter";
 
 const { width, height } = Dimensions.get( "screen" );
 const selectedImageHeight = height - 350;
```

Comment:
I know this isn't a part of this PR, but this is another literal that could probably be calculated from styles or made a constant or something. Usually when I see something like this I think, "Why 350? What's this going to look like on a tablet?"